### PR TITLE
Introduce volume structs

### PR DESCRIPTION
**Pull request by orlandohohmeier**
These structs they will be used to provide a nice API to deal with service volumes.

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @MatApple

File: src/js/structs/VolumeList.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import List from './List';
+import Volume from './Volume';
+
+class VolumeList extends List {}
```
Comment (aldipower) [inline src/js/structs/VolumeList.js]: Here's a semi-colon missing.

Comment (orlandohohmeier) [inline src/js/structs/VolumeList.js]: No, it's a class and there is no need for a semi-colon.

**Comment by orlandohohmeier** [comment]
@mesosphere-ci retest this please

File: src/js/structs/Volume.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
+import Item from './Item';
+import VolumeStatus from '../constants/VolumeStatus';
+
+class Volume extends Item {
+
+  getContainerPath() {
```
Comment (kennyt) [inline src/js/structs/Volume.js]: i think we agreed on using `get containerPath()` yes?

Comment (orlandohohmeier) [inline src/js/structs/Volume.js]: We can't use proper getters yet, as they interfere with the way our `Item` struct is working. Let's add this to our agenda and schedule some time to fix the item struct.

Comment (kennyt) [inline src/js/structs/Volume.js]: ahhh i see what you mean. alright sounds good

**Comment by ssk2** [comment]
retest this please

**Comment by ssk2** [comment]
retest this please

**Pull request merged**